### DISCUSSION TITLE
Discord Server now online

### DISCUSSION THREAD

**Discussion by itsToggle** [itsToggle/plex_debrid#133] [Announcements]
Hey guys, 

Since some of you mentioned that a discord server would be nice, here it is:

https://discord.gg/UKkPeRdukx

Feel free to share your setup, ask about stuff or help others :)

**Comment by RobHaydon92** [comment]
Cannot Join Discord shows: Invite Invalid.

**Reply by itsToggle** [reply]
Ive updated the readme, here it is :) https://discord.gg/UKkPeRdukx

**Comment by k33py** [comment]
Is there a new invite link please

**Reply by mrizvi66** [reply]
yes i can't join the discord

**Reply by itsToggle** [reply]
The invite should still work, it's still listed as active. Anyways, here's a different one: https://discord.gg/u3vTDGjeKE

**Reply by k33py** [reply]
I'm in thanks

**Comment by mrizvi66** [comment]
it worked! thank you!!

On Wed, Mar 29, 2023 at 7:28 PM itsToggle ***@***.***> wrote:

> The invite should still work, it's still listed as active. Anyways, here's
> a different one: https://discord.gg/u3vTDGjeKE
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/itsToggle/plex_debrid/discussions/133#discussioncomment-5471450>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC3FN4XOLCOCWKM6OSAZQ63W6TATBANCNFSM6AAAAAAQQ6ZPDA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Thanks,

Mohammad
